### PR TITLE
Start Position Feature

### PR DESCRIPTION
**Pull request by 63BeetleSmurf**
### Start Position
I much prefer having my notes in the top right of my screen, so have added an option to allow you to choose which corner of the screen new notes are created. This also detects which direction to create new notes, so if you move a note to another corner, new notes will always be created towards the centre of the screen.
### Position New Notes Relative To Parent
If a note is created using the "+" button on another note, the new one is positioned relative to it's "parent" note. Also the positioning has been adjusted to make grabbing the header on a behind note easier.

### CODE REVIEW

**Comment by collinss** [comment]
Hi @63BeetleSmurf sorry it's taken so long to get back to you on this. I didn't forget about it, but I'm of 2 minds about the setting for new note placement. It's not that it's a bad feature or anything, but I do have a couple of concerns about it.

First, is a general problem that all software seems to suffer from over time, and that is feature creep. The more settings we add, the more burden it adds, both for the developers and especially the users. If the feature adds a significant enough benefit to a large enough number of users, then it is worth the cost, especially if the defaults are chosen well, but the less useful the feature, or the fewer people it benefits, the more the costs begin to outweigh the benefits. Now, I'm not saying that I think that's the case here, but I don't know how many users this will benefit, so I'm hesitant.

My second concern is how to present it to users. I definitely agree that positioning a new note near the parent when pressing the '+' is an improvement and should be standard behavior, so how do we let the user know that the option they're configuring only applies when they're creating a note from the status icon or the manager? I've found that a confusing setting is usually more frustrating than no setting at all - especially if you expect it to work one way and it works another or when the setting doesn't seem to apply everywhere. Again, I'm not saying we shouldn't have the option, but I'm hesitant about it.

In conclusion (sorry for the wall of text, I can be rather verbose sometimes), what I'd like to do is merge your first commit, and then have some more discussion at some point about whether to go with (and how to implement) the second.

**Comment by 63BeetleSmurf** [comment]
Hi @collinss,

I had written a long reply there but by the end I realised, if starting relative to the parent had already been the standard behaviour before I made the start position option, I don’t think there would have been any concern over it being confusing. Putting together a modification to set an option along with a modification that will ignore said option is a bit jarring.

I agree going with just the relative to parent modification just now and having another look at the start position option at a later date is a good way to go.

**Comment by collinss** [comment]
@63BeetleSmurf could you rebase this on master? I tried to cherry-pick your first commit, but there are some merge conflicts, and it was getting really messy. Or alternatively, you could submit a new pr with just that first commit.

**Comment by 63BeetleSmurf** [comment]
I've rebased it, if that does not work let me know and I'll put in a new pull.

**Comment by collinss** [comment]
That didn't seem to work. Perhaps it would be easier to just open a new pr with just the changes from the first commit. I could have just applied the changes myself directly, but then you wouldn't have gotten credit for it.

**Comment by 63BeetleSmurf** [comment]
Hi @collinss,

I've just redone the modification, as there seems to be some code from my cycle colours mod mixed in with this, and put in a new pull request: https://github.com/linuxmint/sticky/pull/73

**Comment by collinss** [comment]
Thanks, I've just merged the other pull request, so I'm going to go ahead and close this one, but if you'd like to discuss the other part of this pull request further, feel free to do so here, or open a separate issue. Either way, I should see it.

**Comment by AspyMui** [comment]
Hi, I've found myself in a unique situation that I'd like to voice my opinion on this new feature. If possible could we add in a center option to the 4 that this feature has implemented? I have a bit of a unique setup with stacked monitors with my top one being smaller. When I first made a note it appeared off the screen in the top left where I had no visible screen area. I figured this out by moving the notes manually around in the saved file location. Center location I believe would guarantee it loads on any weird monitor layout.

Below is my monitor layout for reference.
Screenshot from 2022-12-19 13-24-12
